packages/runtime/container-runtime/src/blobManager/blobManager.ts

```diff
@@ -256,11 +262,11 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
 					return;
 				}
 			}
-
+			this.pendingStashedBlobs.set(localId, this.uploadBlob(localId, blob));
```

Comment:
this never gets cleaned up. i wonder if it would be better to just add an wasStashed property to the existing pending blob, and filter on that, rather than managing another map.

Reply:
the existing pending blob will get removed by the time this uploadBlob promises resolved so we need to keep track of them separately. It's true we don't clean them, I will think on a safe way to do it.